proposals/pravega.adoc

```diff
@@ -0,0 +1,214 @@
+
+A Pravega cluster consists of controllers, segment stores, long-term storage, BookKeeper bookies, and a ZooKeeper ensemble. Pravega deploys natively on Kubernetes with a set of operators maintained by Pravega: a Pravega operator, a BookKeeper operator and a ZooKeeper operator.
+
+The controllers manage the lifecycle of streams, while the segment stores are responsible for persisting segment data and managing the lifecycle of segments. The controllers command the segment store to create, delete, merge, and seal segments as part of their operations.
+
+The segment store uses Apache BookKeeper ledgers to implement journals. Such journals guarantee the durability of appended data, providing both low latency and high throughput. Data written to BookKeeper remains in a memory cache until it is flushed to long-term storage, which happens asynchronously. Long-term storage is expected to be a horizontally scalable storage system, which is a primary characteristic of cloud storage. Pravega can use both cloud storage based on files or objects.
+
+Apache ZooKeeper is used for coordination tasks, such as the assigning and balancing of work across segment stores, but not storing stream metadata, which we store in Pravega itself using a key-value pair API implemented on top of segments.
+
+See more detail about Pravega in the documentation on the http://pravega.io[Pravega Web site]
+
+=== CNCF Storage SIG presentation
+
+Our presentation to the Storage SIG has already been recorded and is available on YouTube:
+
+https://www.youtube.com/watch?v=AZX0LQwGB9E&list=PLj6h78yzYM2NoiNaLVZxr-ERc1ifKP7n6&t=11m29s
+
+=== Conference presentations and Podcasts
+
+* https://bigdatabeard.com/streaming-storage-reimagined/[Big Data Beard]
+* https://softwareengineeringdaily.com/2020/05/07/pravega-storage-for-streams-with-flavio-junquiera/[Software Engineering Daily] 
+* Flink Forward 2020:
+** https://www.youtube.com/watch?v=_xaf6ICDrg4
+** https://www.youtube.com/watch?v=av2FJWK2xik
+* https://www.cncf.io/webinars/pravega-rethinking-storage-for-streams/[CNCF webinar 2020]
+* https://www.microsoft.com/en-us/research/video/pravega-a-new-storage-abstraction-data-streams/[Microsoft Research]
+* Flink Forward Asia 2019
+** https://www.youtube.com/watch?v=SEyYrQx9sJU
+** https://www.youtube.com/watch?v=g8Kcle3kAws
+* https://www.youtube.com/watch?v=abNulm1P6M0[DataWorks Barcelona 2019]
+* https://conferences.oreilly.com/strata/strata-eu-2018/public/schedule/detail/65388[Strata London 2018]
+* https://www.youtube.com/watch?v=GEpdZA1eyS4[DataWorks Summit 2018]
+* https://conferences.oreilly.com/strata/strata-ca-2018/public/schedule/detail/63888[Strata San Jose 2018]
+* https://www.youtube.com/watch?v=FstZzVUVAdE[Flink Forward San Francisco 2018]
+
+=== Comparison with NATS
```

Comment:
comparison with Kafka would've been useful

Reply:
Thanks for showing interest in the proposal, @yurishkuro. We did not compare with Apache Kafka because we understood that the interest would be on existing CNCF projects. We can certainly add information or simply respond as a comment. Let us know how you want us to address it and if there is any other information you are after other than general curiosity.

Reply:
I am not on the TOC so my opinion is not a requirement for you to change the proposal, but looking back at other projects, usually the question is about "how do you compare with other competing open source projects", not whether those projects are in the CNCF. As you said, NATS is much further away from Pravega than projects like RabitMQ or Kafka.

Reply:
Understood, we will either add a comparison to the proposal or point to one.

Reply:
what about comparison with nats-streaming, which is closer to kafka

Reply:
@nuharaf -- Or, better yet, a `pravegastore` implementation for nats-streaming here: https://github.com/nats-io/nats-streaming-server/tree/master/stores :)

@yurishkuro -- I wrote a comparison with Kafka as an appendix to the proposal. I should have it committed ASAP. The comparison is undergoing final review today.